src/control/KinematicSimulationTrajectoryExecutor.cpp

```diff
@@ -106,7 +106,7 @@ void KinematicSimulationTrajectoryExecutor::step()
   }
 
   auto timeSinceBeginning = system_clock::now() - mExecutionStartTime;
-  double t = duration_cast<seconds>(timeSinceBeginning).count();
+  auto t = duration<double>(timeSinceBeginning).count();
```

Comment:
Why was this changed to `duration` instead of `duration_cast?` I believe `timeSinceBeginning` is a duration, so `duration_cast` was appropriate here. 🤔